pkg/security/podsecuritypolicy/group/mustrunas_test.go

```diff
@@ -111,23 +114,33 @@ func TestValidate(t *testing.T) {
 		ranges []extensions.GroupIDRange
 		groups []int64
 		pass   bool
+		errs   field.ErrorList
 	}{
 		"nil security context": {
 			ranges: []extensions.GroupIDRange{
 				{Min: 1, Max: 3},
 			},
+			errs: field.ErrorList{
+				field.Invalid(field.NewPath(""), []int64(nil), "unable to validate empty groups against required ranges"),
+			},
 		},
 		"empty groups": {
 			ranges: []extensions.GroupIDRange{
 				{Min: 1, Max: 3},
 			},
+			errs: field.ErrorList{
+				field.Invalid(field.NewPath(""), []int64(nil), "unable to validate empty groups against required ranges"),
+			},
 		},
 		"not in range": {
 			groups: []int64{5},
 			ranges: []extensions.GroupIDRange{
 				{Min: 1, Max: 3},
 				{Min: 4, Max: 4},
 			},
+			errs: field.ErrorList{
+				field.Invalid(field.NewPath(""), []int64{5}, "group 5 must be in the ranges: [{1 3} {4 4}]"),
```

Comment:
The message isn't perfect but better than it was. I won't insist on making the ranges more readable, maybe you or other reviewers decide that it's required.

BTW @dmage WDYT do we need to further improve it?